modules/core/src/geometry/geometry.js

```diff
@@ -0,0 +1,119 @@
+import {uid, assert} from '../utils';
```

Comment:
This file used to exist elsewhere, and it doesn't look like the other file was deleted.

Reply:
Also, there are now separate directories for `geometry` and `geometries` perhaps they should be consolidated?

Reply:
Fine if we would like to do so in a later PR. I tend to want keep the basic "lib" class and the library of instances separate. The reason geometry doesn't go into core is that this is a lowe layer that does not have knowledge of webgl...